default/web_tt2/skinsedit.tt2

```diff
@@ -1,459 +1,462 @@
+<!-- skinsedit.tt2 -->
 <!-- $Id: skinsedit.tt2 5649 2009-04-07 14:33:38Z serge.aumont -->
```

Comment:
This comment line including Subversion ID should be removed.

Reply:
Fixed in https://github.com/sympa-community/sympa/pull/582/commits/7b3a39c6ac7c013a0990a7033f81118d1943e2de (and in lists_categories.tt2 as well).

Reply:
Confirmed.